package.json

```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/AugurProject/augur"
   },
   "scripts": {
+    "clean": "rm -rf build; rm -rf node_modules",
```

Comment:
AFAIK, this will fail when run in a win environment.

Ideally use something like `rimraf` for compatibility.

Reply:
updated to use rimraf, great catch.